### DISCUSSION TITLE
After Install Lunar package then when login hub then this issue occur.How solve this.Plz Help

### DISCUSSION THREAD

**Discussion by SAINI673** [lunarphp/lunar#984] [Q&A]
Lunar\DataTypes\Price::__construct(): Argument #2 ($currency) must be of type Lunar\Models\Currency, null given, called in E:\Sandeep-vishal_project\e_shop\vendor\lunarphp\lunar\packages\core\src\Base\Casts\Price.php on line 39

**Comment by glennjacobs** [comment]
Do you have any currencies set in the database?
